docs/source/projects.rst

```diff
@@ -87,6 +88,29 @@ Conda environment
   file in the root of the project directory or by including a ``conda_env`` entry in your
   ``MLproject`` file. For details, see the :ref:`project-directories` and :ref:`mlproject-specify-environment` sections.
 
+  The ``mlflow run`` command supports running a Conda environment project as a Virtualenv environment project.
```

Comment:
Nit: `conda` is also lower-case; https://docs.conda.io/en/latest/

Reply:
yep!

Reply:
@dbczumar @BenWilson2 Lower-cased